paper-icon-button.html

```diff
@@ -76,7 +76,11 @@
         position: relative;
         padding: 8px;
         outline: none;
-        -webkit-tap-highlight-color: rgba(0,0,0,0);
+
+        /* NOTE: Both values are needed, since some phones require the value to be `transparent`. */
```

Comment:
Ok - I need to update this change in paper-checkbox too then ([here's a PR I merged yesterday](https://github.com/PolymerElements/paper-checkbox/pull/127)).